core/lib/env_config/src/via_wallets.rs

```diff
@@ -17,18 +17,26 @@ impl FromEnv for ViaWallets {
     fn from_env() -> anyhow::Result<Self> {
         let wallets = Wallets::from_env()?;
 
+        let btc_sender_address = pk_from_env(
```

Comment:
`pk_from_env` name looks a bit misleading since it is used to load the addresses as well, but no need to change it now, let improve in the future.